### ISSUE TITLE
Elevate launcher process when doing patching

### ISSUE THREAD

**Issue by SonnyX** [TotemArts/Launcher#38]
![afbeelding](https://user-images.githubusercontent.com/784475/86415010-fc8e8800-bcc5-11ea-8548-a18204ba7741.png)

**Comment by SonnyX** [comment]
```powershell
($acl = Get-ACL).AddAccessRule((New-Object System.Security.AccessControl.FileSystemAccessRule([System.Security.Principal.WindowsIdentity]::GetCurrent().Name,"FullControl","Allow"))); $acl | Set-ACL
```

**Comment by SonnyX** [comment]
fixed in 118095f
